order/src/main/java/com/caldevsupplychain/order/service/OrderServiceImpl.java

```diff
@@ -83,38 +135,25 @@ public OrderBean deleteOrder(OrderBean orderBean) {
 	}
 
 	@Override
-	public Optional<List<OrderBean>> findByDisplayId(String displayId) {
-		List<Order> orders = orderRepository.findByDisplayId(displayId, new PageRequest(0, Integer.MAX_VALUE));
-		if(orders != null){
-			return Optional.of(orderMapper.ordersToBeans(orders));
-		}
-		return Optional.empty();
-	}
+	public Optional<List<OrderBean>> findByUserUuid(String userUuid) {
```

Comment:
Don't wrap Optional around a list. Always use list.isEmpty() to determine if a list is empty or not.

Reply:
Fixed